auth-web/src/components/auth/manage-business/AffiliatedEntityTable.vue

```diff
@@ -358,14 +375,33 @@ export default class AffiliatedEntityTable extends Mixins(DateMixin) {
         }
       }
     }
-    const filingResponse = await this.createNamedBusiness(filingBody)
+    return this.createNamedBusiness(filingBody, business.businessIdentifier)
+  }
 
-    if (filingResponse?.errorMsg) {
-      this.$emit('add-unknown-error')
-      return ''
-    } else {
-      return filingResponse.data.filing.business.identifier
```

Comment:
Will this still handle a failed creation? 
`createNamedBusiness` will only return an error message it looks like. Looks like it was handled on here previously.

Reply:
I see, you moved it up 1 level.